tx_test.go

```diff
@@ -202,7 +202,7 @@ func TestTx_CreateTx(t *testing.T) {
 		2000000)
 	assert.NoError(t, err)
 
-	err = tx.PayTo("n2wmGVP89x3DsLNqk3NvctfQy9m9pvt7mk", 1999942)
+	err = tx.PayToAddress("n2wmGVP89x3DsLNqk3NvctfQy9m9pvt7mk", 1999942)
```

Comment:
were these tests failing previously?

Reply:
oh nevermind - seems we didn't port over the `PayToAddress` func

Reply:
I just changed PayTo to take a script and then the original PayTo I renamed to PayToAddress